sdk-tests/src/test/java/io/dapr/it/testcontainers/workflows/crossapp/WorkflowsCrossAppCallActivityIT.java

```diff
@@ -0,0 +1,278 @@
+        .withWorkingDirectory("/app")
+        .withCommand("java", "-cp", "/app/classes:/app/libs/*", 
+                     "-Ddapr.app.id=app2",
+                     "-Ddapr.grpc.endpoint=app2-sidecar:50001",
+                     "-Ddapr.http.endpoint=app2-sidecar:3500",
+                     "io.dapr.it.testcontainers.workflows.crossapp.App2Worker")
+        .withNetwork(DAPR_NETWORK)
+        .waitingFor(Wait.forLogMessage(".*App2Worker started.*", 1))
+        .withLogConsumer(outputFrame -> System.out.println("App2Worker: " + outputFrame.getUtf8String()));
+
+    // Start app3 worker (connects to APP3_CONTAINER)
+    app3Worker = new GenericContainer<>("openjdk:17-jdk-slim")
+        .withCopyFileToContainer(MountableFile.forHostPath("target/test-classes"), "/app/classes")
+        .withCopyFileToContainer(MountableFile.forHostPath("target/dependency"), "/app/libs")
+        .withWorkingDirectory("/app")
+        .withCommand("java", "-cp", "/app/classes:/app/libs/*", 
+                     "-Ddapr.app.id=app3",
+                     "-Ddapr.grpc.endpoint=app3-sidecar:50001",
+                     "-Ddapr.http.endpoint=app3-sidecar:3500",
+                     "io.dapr.it.testcontainers.workflows.crossapp.App3Worker")
+        .withNetwork(DAPR_NETWORK)
+        .waitingFor(Wait.forLogMessage(".*App3Worker started.*", 1))
+        .withLogConsumer(outputFrame -> System.out.println("App3Worker: " + outputFrame.getUtf8String()));
+    
+    // Start all worker containers
+    crossappWorker.start();
+    app2Worker.start();
+    app3Worker.start();
+  }
+
+  @AfterAll
+  public static void tearDown() {
+    if (crossappWorker != null) {
+      crossappWorker.stop();
+    }
+    if (app2Worker != null) {
+      app2Worker.stop();
+    }
+    if (app3Worker != null) {
+      app3Worker.stop();
+    }
+    if (MAIN_WORKFLOW_CONTAINER != null) {
+      MAIN_WORKFLOW_CONTAINER.stop();
+    }
+    if (APP2_CONTAINER != null) {
+      APP2_CONTAINER.stop();
+    }
+    if (APP3_CONTAINER != null) {
+      APP3_CONTAINER.stop();
+    }
+    if (sharedPlacementContainer != null) {
+      sharedPlacementContainer.stop();
+    }
+    if (sharedSchedulerContainer != null) {
+      sharedSchedulerContainer.stop();
+    }
+  }
+
+  @Test
+  public void testCrossAppWorkflow() throws Exception {
+    // TestContainers wait strategies ensure all containers are ready before this test runs
+
+    String input = "Hello World";
+    String expectedOutput = "HELLO WORLD [TRANSFORMED BY APP2] [FINALIZED BY APP3]";
+    
+    // Create workflow client connected to the main workflow orchestrator
+    // Use the same endpoint configuration that the workers use
+    // The workers use host.testcontainers.internal:50001, so we need to use the mapped port
+    String grpcEndpoint = "localhost:" + MAIN_WORKFLOW_CONTAINER.getMappedPort(50001);
+    String httpEndpoint = "localhost:" + MAIN_WORKFLOW_CONTAINER.getMappedPort(3500);
```

Comment:
Could we please remove the `System.setProperty()` calls, this have caused a lot of issues in the past. As far as I remember we are using property overrides everywhere. I think property overrides should be enough.